source/features/locked-issue-indicator.tsx

```diff
@@ -0,0 +1,60 @@
+import './locked-issue-indicator.css';
+
+import React from 'react';
+import {LockIcon} from '@primer/octicons-react';
+import * as pageDetect from 'github-url-detection';
+
+import {$} from 'select-dom';
```

Comment:
The space is only needed between `node_modules` and local imports
```suggestion
import './locked-issue-indicator.css';
import React from 'react';
import {LockIcon} from '@primer/octicons-react';
import * as pageDetect from 'github-url-detection';
import {$} from 'select-dom';
```